src/ansys/units/cfg.yaml

```diff
@@ -9,12 +9,12 @@ dimension_order:
   Length: 2
   Time: 3
   Temperature: 4
-  Temperature Difference: 4
-  Angle: 5
-  Chemical Amount: 6
-  Light: 7
-  Current: 8
-  Solid Angle: 9
+  Temperature Difference: 5
```

Comment:
Yes, I think that this is a necessary change.

Reply:
@ansaboutin it should be useful to exactly align the names here with those in the enum in dimensions.py. 
See https://docs.python.org/3/howto/enum.html wherein:
```
>>> Color['RED']
<Color.RED: 1>
>>> Color['GREEN']
<Color.GREEN: 2>
```

Reply:
...actually, this is redundant now

Reply:
... See stuff like this further on down:
```
fundamental_units:
  kg:
    type: Mass
    factor: 1
    offset: 0
```
`type: Mass` can change to `type: MASS` to align exactly with the enum key.